### PR TITLE
Move runtimes from AutoRest repo to here

### PR DESCRIPTION
**Pull request by vishrutshah**
```
Azure-sdk-for-ruby
      |--- runtime
            |--- ms-rest
                |--- lib
                |--- …
                |--- ms_rest.gemspec
            |--- ms-rest-azure
                |--- lib
                |--- ...
                |--- ms_rest_azure.gemspec
```
#505

### CODE REVIEW

**Comment by vishrutshah** [comment]
@veronicagg & @sarangan12 Feel free to review this PR when you get a chance. Thanks!

File: runtime/ms_rest/ms_rest.gemspec [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+Gem::Specification.new do |spec|
+  spec.name          = 'ms_rest'
+  spec.version       = MsRest::VERSION
+  spec.authors       = 'Microsoft Corporation'
+  spec.email         = 'azsdkteam@microsoft.com'
+
+  spec.summary       = %q{Azure Client Library for Ruby.}
+  spec.description   = %q{Azure Client Library for Ruby.}
+  spec.homepage      = 'https://aka.ms/azure-sdk-for-ruby'
```
Comment (vishrutshah) [inline runtime/ms_rest/ms_rest.gemspec]: updated to https://aka.ms/ms_rest

File: runtime/ms_rest_azure/ms_rest_azure.gemspec [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+Gem::Specification.new do |spec|
+  spec.name          = 'ms_rest_azure'
+  spec.version       = MsRestAzure::VERSION
+  spec.authors       = 'Microsoft Corporation'
+  spec.email         = 'azsdkteam@microsoft.com'
+
+  spec.summary       = %q{Azure Client Library for Ruby.}
+  spec.description   = %q{Azure Client Library for Ruby.}
+  spec.homepage      = 'https://aka.ms/azure-sdk-for-ruby'
```
Comment (vishrutshah) [inline runtime/ms_rest_azure/ms_rest_azure.gemspec]: updated to https://aka.ms/ms_rest_azure

File: runtime/ms_rest/ms_rest.gemspec [hunk 3]
```diff
@@ -14,7 +14,7 @@ Gem::Specification.new do |spec|
 
   spec.summary       = %q{Azure Client Library for Ruby.}
   spec.description   = %q{Azure Client Library for Ruby.}
-  spec.homepage      = 'https://aka.ms/azure-sdk-for-ruby'
+  spec.homepage      = 'https://aka.ms/ms_rest'
```
Comment (veronicagg) [inline runtime/ms_rest/ms_rest.gemspec]: Are these shortcuts created? They don't seem to be working right now.

**Comment by vishrutshah** [comment]
Thanks for the review!!

**Pull request merged**